beginner_source/translation_transformer.py

```diff
@@ -0,0 +1,399 @@
+"""
+Language Translation with Transformer
+=====================================
+**Author**: `Jayasimha T <https://github.com/iamsimha>`_
+
+This tutorial shows, how to train a translation model from scratch using
+Transformer. We will be using `Multi30k
```

Comment:
Are we going to move `multi30k` dataset to the root folder in 0.10.0 release. If so, we should remember to update this tutorial and switch to the one in torchtext. cc @parmeet

Reply:
As the follow-up, I don't think we need to mention this  github repo here.